pkg/flagbuilder/build_flags.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Are you moving this file around or is it new code?  If this is new ... unit test pretty please.